rust_dev_preview/apigatewaymanagement/README.md

```diff
@@ -0,0 +1,22 @@
+# AWS SDK for Rust code examples for API Gateway
+
+The Amazon API Gateway Management API allows you to directly manage runtime aspects of your deployed APIs. To use it,
+you must explicitly set the SDK's endpoint to point to the endpoint of your deployed API. The endpoint will be of the
+form `https://[api-id].execute-api.[region].amazonaws.com/[stage]` where:
+* `api-id` is the ID of your API (eg. `xy4n5r0m12`)
+* `region` is the region of your API (eg. `us-east-1`)
+* `stage` is the deployment stage of your API (eg. `test`, `prod`, `beta`)
```

Comment:
Style guide says avoid 'e.g.'. I'd just take out them out altogether because they're not really necessary.

Reply:
Removed